src/python/goal-py.c

```diff
@@ -411,8 +414,10 @@ run_all(_GoalObject *self, PyObject *args, PyObject *kwds)
 	return NULL;
 
     struct _PySolutionCallback cb_s = {callback_tuple, callback, 0};
-    int ret = hy_goal_run_all_flags(self->goal, py_solver_callback, &cb_s,
-				    flags);
+    int ret;
+    Py_BEGIN_ALLOW_THREADS;
```

Comment:
this causes tests to fail. It takes python function callback and segfaults in hawkey python tests.